src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -702,4 +701,23 @@ class SlickDataAccess(databaseConfig: Config) extends DataAccess {
       })
     }
   }
+
+  override def updateCallCaching(parameters: CallCachingParameters): Future[Int] = {
+    // Figure out which of the three possible queries to use based on whether a call has been specified and
+    // if so whether an index has been specified.
+    val executionQuery: (Int) => Query[SlickDataAccess.this.dataAccess.Executions, Execution, Seq] = {
```

Comment:
That's the ascription as generated by IntelliJ.  The `SlickDataAccess.this` bit seems unnecessary, this is a reference to a class defined within a trait implemented by `dataAccess: DataAccessComponent`.